src/main/java/com/ericsson/ec/JenkinsManager.java

```diff
@@ -349,17 +351,19 @@ public boolean installPlugin(String plugin, String version) throws Exception {
 
         String scriptData = "<jenkins><install plugin='" + plugin + "@" + version + "' /></jenkins>";
 
-        httpRequest.setBaseUrl(jenkinsBaseUrl).addHeader("Authorization", "Basic " + encoding)
-                .addHeader("Content-type", MediaType.TEXT_XML).addHeader("Jenkins-Crumb", crumb).setBody(scriptData)
-                .setEndpoint("/pluginManager/installNecessaryPlugins");
+        httpRequest.setBaseUrl(jenkinsBaseUrl)
+                   .addHeader("Authorization", "Basic " + encoding)
+                   .addHeader("Content-type", MediaType.TEXT_XML)
+                   .addHeader("Jenkins-Crumb", crumb)
+                   .setBody(scriptData)
+                   .setEndpoint("/pluginManager/installNecessaryPlugins");
 
         ResponseEntity response = httpRequest.performRequest();
         success = response.getStatusCode() == HttpStatus.SC_MOVED_TEMPORARILY;
 
         if (!success) {
-            String message = "Failed to add a plugin with name " + plugin + " and version " + version
-                    + " to Jenkins. Response code: " + response.getStatusCodeValue() + " and body: "
-                    + response.getBody();
+            String message = "Failed to add a plugin with name " + plugin + " and version " + version + " to Jenkins. Response code: "
+                    + response.getStatusCodeValue() + " and body: " + response.getBody();
             throw new Exception(message);
         }
```

Comment:
Do we check if Plugin is successfully installed and usable when this function has returned?
It is no meaning to execute tests if required plugin is not installed and/or is not working.

Reply:
I did not write this. It checks a http status code after performing a request which is used to check success.

Reply:
there is a function to lookup installed plugins, the test should verify installation by using that function.